src/firstboot/firstboot.c

```diff
@@ -285,6 +288,80 @@ static int process_locale(void) {
         return 0;
 }
 
+static int prompt_keymap(void) {
+        _cleanup_strv_free_ char **kmaps = NULL;
+        int r;
+
+        if (arg_keymap)
+                return 0;
+
+        if (!arg_prompt_keymap)
+                return 0;
+
+        r = get_keymaps(&kmaps);
```

Comment:
I figure we should add some additional handlig here, if no keymaps are installed, and simply skip the operation in that case